cumulusci/cumulusci.yml

```diff
@@ -126,6 +126,16 @@ tasks:
             children_only: True
             source_branch: $project_config.repo_branch
         group: GitHub
+    github_publish_release:
```

Comment:
What about calling it "clone release"?

Reply:
Actually, thinking about this more and `github_clone_release` might be confusing since clone has specific meaning in git.  Perhaps `github_copy_release`?

Reply:
@jlantz yeah ok

Reply:
@jlantz although we do have an existing github_clone_tag task